docs/source/reproducers/03-zuul.md

```diff
@@ -137,6 +145,35 @@ $ tail -f ~/ansible.log
 
 Note that `tmux` is installed by default, and may be handy in such a case ;).
 
+#### Zuul_inventory.yml
+That file is a generated inventory, allowing ansible to know about the compute(s) and CRC nodes.
+
+It exposes each node with the connection information (user and IP), and also created groups in
+order to access, for instance, all of the computes. Those groups mimic the same behaviour we
+have in Zuul.
+
+### Ansible tags of interest
+You may want to re-run some parts of the reproducer only, and avoid re-running the whole
+virtual machine creation.
+
+In order to do so, the reproducer exposes two tags:
+* bootstrap_layout
+* bootstrap
+
+#### Tag: bootstrap
+This covers everything from the creation of the virtual layout (networks and machines) to
+the push of generated plays. With that, you'll only get the repositories, and some of the
+zuul data, mostly.
+
+Use with caution, since it may end with a broken environment, where some files may miss an update,
```

Comment:
Do you mean `Skip with caution`? Does it mean `bootstrap` is required to download updates?

Reply:
reworded